### ISSUE TITLE
Incorrect images in documentation

### ISSUE THREAD

**Issue by mmarcinkiewicz** [apache/incubator-mxnet#19583]
## Description
The same image (cosine) is used to show the different options for the Learning Rate Decay.

### Error Message
none

## To Reproduce
---

### Steps to reproduce
https://mxnet.apache.org/versions/1.7.0/api/python/docs/tutorials/packages/gluon/training/learning_rates/learning_rate_schedules.html

## What have you tried to solve it?
---

## Environment
---

<details>
![image](https://user-images.githubusercontent.com/43240942/100139167-5834fa00-2e8f-11eb-8673-eebc608f0953.png)

**Comment by mmarcinkiewicz** [comment]
The image is not being shown
![image](https://user-images.githubusercontent.com/43240942/100139262-7e5a9a00-2e8f-11eb-98fb-99c178585ad1.png)

**Comment by bkmgit** [comment]
The images are correct in the version available at https://github.com/apache/incubator-mxnet/blob/v1.7.x/docs/python_docs/python/tutorials/packages/gluon/training/learning_rates/learning_rate_schedules.md but not on the deployed site https://mxnet.apache.org/versions/1.7.0/api/python/docs/tutorials/packages/gluon/training/learning_rates/learning_rate_schedules.html maybe the site needs to be re-generated?
